algorithms/kernel/pca/oneapi/pca_dense_correlation_online_kernel_ucapi_impl.i

```diff
@@ -124,7 +124,11 @@ Status PCACorrelationKernelOnlineUCAPI<algorithmFPType>::finalize(PartialResult<
     }
 
     data_management::NumericTablePtr correlation = parameter->covariance->getResult()->get(covariance::covariance);
-    DAAL_CHECK_STATUS_VAR(_host_impl->computeCorrelationEigenvalues(*correlation, eigenvectors, eigenvalues));
+
+    {
+        DAAL_ITTNOTIFY_SCOPED_TASK(compute.full.computeEigenvalues);
```

Comment:
nip: `full` means that it is related to the covariance method, when the covariance matrix is calculated inside PCA instead of being provided as input. But eigenvalues are calculated for both branches. Suggession: change it for `compute.covariance.computeEigenvalues` or `compute.defaultDense.computeEigenvalues`